engines/dfc_provider/lib/dfc_provider/address.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Temporary solution to add `region` to Address, to be removed once the DFC connector supports it
+
+module DfcProvider
+  class Address < DataFoodConsortium::Connector::Address
+    # @return [String]
+    attr_accessor :region
+
+    def initialize(semantic_id, region: "", **properties)
```

Comment:
I'm planning to set all attributes to `nil` by default instead of the empty string. That will allow us to know if an imported document contained an attribute or if it was completely absent. It doesn't really matter here but I thought I let you know.